app/screens/HomeScreen.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+import LoginScreen from './auth/login/LoginScreen';
+
+export default class HomeScreen extends Component {
+    constructor(props) {
+        super(props);

```

Comment:
you are not using the constructor, so you can do a functional component, anyway I think that homescreen is not necessary, you can use login as you main screen.